libs/services/src/events/hatom.events.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from "@nestjs/common";
+import { EventLog } from "apps/api/src/endpoints/events/entities";
+import { Address } from "@multiversx/sdk-core";
+import BigNumber from "bignumber.js";
+import { CsvRecordsService } from "../records";
+import moment from "moment";
+import { DataService } from "../data";
+import { TableSchema } from "apps/dune-simulator/src/endpoints/dune-simulator/entities";
+
+interface BorrowEvent {
+    eventName: string;
+    borrowerAddress: string;
+    amount: BigNumber;
+    newAccountBorrow: BigNumber;
+    newTotalBorrows: BigNumber;
+    newBorrowerIndex: BigNumber;
+}
+
+@Injectable()
+export class HatomEventsService {
+    private readonly headers: TableSchema[] = [
+        { name: 'borrower_address', type: 'varchar' },
+        { name: 'timestamp', type: 'varchar' },
+        { name: 'borrowed_amount', type: 'double' },
+        { name: 'borrowed_amount_in_egld', type: 'double' },
+        { name: 'borrowed_amount_in_usd', type: 'double' },
+        { name: 'total_borrowed', type: 'double' },
+        { name: 'account_borrowed', type: 'double' },
+        { name: 'borrowed_token', type: 'varchar' },
+    ];
+    constructor(
+        private readonly csvRecordsService: CsvRecordsService,
+        private readonly dataService: DataService,
+    ) { }
+
+    public async hatomWebhook(eventsLog: EventLog[], borrowedToken: string): Promise<void> {
+
+        for (const eventLog of eventsLog) {
+            const borrowEventInHex = '626f72726f775f6576656e74';
```

Comment:
you could have defined the const as regular string and decode the first topic. easier to see what we're looking for